Jenkinsfile

```diff
@@ -129,6 +136,28 @@ set -o pipefail
 make clean distclean
 export ENGBLD_BITS_UPLOAD_IMGAPI=true
 make print-BRANCH print-STAMP triton-origin-x86_64-21.4.0-"buildimage bits-upload"
+''')
+            }
+        }
+        stage('triton-origin-image-x86_64-22.4.0') {
+            agent {
+                node {
+                    label joyCommonLabels(image_ver: '22.4.0',
```

Comment:
22.4 images should be built from `image_ver: '21.4.0'` due to circular dependency issue.

Context: The Jenkinsfile was added after 19.4 already existed, so this circular dependency was never encountered at Joyent. When I created the 21.4.0 image I did it manually so also didn't have to deal with this.

Reply:
I made this change and the build on the branch completed fine on 21.4.